### PR TITLE
Cleanup references

### PR DESCRIPTION
**Pull request by djbe**
Fixes https://github.com/SwiftGen/Eve/issues/3.
Together with https://github.com/SwiftGen/templates/pull/23

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # SwiftGen
 
-[![Build Status](https://travis-ci.org/SwiftGen/SwiftGen.svg?branch=master)](https://travis-ci.org/AliSoftware/SwiftGen)
+[![Build Status](https://travis-ci.org/SwiftGen/SwiftGen.svg?branch=master)](https://travis-ci.org/SwiftGen/SwiftGen)
```
Comment (AliSoftware) [inline README.md]: ?! I was quite sure to have fixed that badge in the great split branch?

Comment (djbe) [inline README.md]: the first bit yeah, the url no?

Comment (AliSoftware) [inline README.md]: Oh. 🙃

**Comment by AliSoftware** [comment]
I'm wondering if we should fix the links in the CHANGELOG for past versions sections? (so that CHANGELOG links will be keep being valid even if GitHub stops the redirection at some point)?

**Pull request merged**